sdk/personalizer/azure-ai-personalizer/src/main/java/com/azure/ai/personalizer/models/ErrorResponse.java

```diff
@@ -5,20 +5,31 @@
 package com.azure.ai.personalizer.models;
 
 import com.azure.core.annotation.Fluent;
-import com.fasterxml.jackson.annotation.JsonProperty;
+import com.azure.json.JsonReader;
+import com.azure.json.JsonSerializable;
+import com.azure.json.JsonToken;
+import com.azure.json.JsonWriter;
+import java.io.IOException;
 
-/** Used to return an error to the client. */
+/**
+ * Used to return an error to the client.
+ */
 @Fluent
-public final class ErrorResponse {
+public final class ErrorResponse implements JsonSerializable<ErrorResponse> {
```

Comment:
This is outside the scope of this pr but we should not have these error types and instead use ResponseError from azure-core.

Reply:
Agreed, should open an issue on the generator. There is one concern with doing that with the error type being extended in the future (but I don't think I've ever seen that)